package.json

```diff
@@ -3,9 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "^1.10.0",
+    "jest-enzyme": "^7.0.1",
```

Comment:
Nice job you did here. Is there any reason you may have had as to why you decided to use `jest-enzyme` when you already installed enzyme. Don't you think it would have been a better approach if you had gone with installing `jest`. Jest comes with a couple of interesting things that we shall need to use in the future and these may include setting up the `snapshotSerializers`. On another note, I don't know what you think about adding `watch` points on the test in scripts

Reply:
@missvicki Thanks very much for this feedback. Now to answer your questions, jest-enzyme will help if we need to write more readable matchers in our tests. You can see some official documentation [here](https://facebook.github.io/create-react-app/docs/running-tests#initializing-test-environment). 
Also, create-react-app comes with jest and features like `watch` mode, `snapshots` are integrated. For example, there is already a running test that uses `snapshot` in `App.test.jsx`

Reply:
Jest is one of the dependencies for react-scripts. https://www.npmjs.com/package/react-scripts
So we already have that.

Reply:
Thanks for this insight.